### ISSUE TITLE
Overview Change

### ISSUE THREAD

**Issue by CookieLightwood** [SethClydesdale/forumactif-edge#53]
Hello!

I’m having a problem, which is that I can’t find a way to change this (which is an image) to a custom one that I want to add. Need to use some HTML code? Notice, I don’t know how to use CSS very well!

[https://i.imgur.com/wgch7vR.png](url)

I want to remove the current image and replace with another!

Thanks!

**Comment by SethClydesdale** [comment]
Hello @CookieLightwood!

Are you referring to the logo? This can be changed under the Display > Images section in the administration panel.

**Comment by CookieLightwood** [comment]
Yes, the logo. However I can't find the Images section from Display :/
![Screenshot_34](https://user-images.githubusercontent.com/79884195/156079592-eefd9a55-798e-4aea-848f-c10a89954683.png)

**Comment by SethClydesdale** [comment]
It's "Pics management" (under pictures and colors)

Sorry, I haven't used Forumactif in awhile so I forgot the exact name haha.
